### PR TITLE
client: Add command to create Java keystore/truststore/client properties

### PR DESCRIPTION
**Pull request by Ormod**
This makes the use of Java based clients much easier as we create
a suitable keystore/truststore/client properties file for immediate
use.

### CODE REVIEW

File: aiven/client/cli.py [hunk 1]
```diff
@@ -519,13 +520,63 @@ def service_user_list(self):
...
+        subprocess.check_call([
+            "openssl", "pkcs12", "-export",
+            "-out", os.path.join(self.args.target_directory, "client.keystore.p12"),
+            "-inkey", os.path.join(self.args.target_directory, "service.key"),
+            "-in", os.path.join(self.args.target_directory, "service.cert"),
+            "-certfile", os.path.join(self.args.target_directory, "ca.pem"),
+            "-passout", "pass:{}".format(self.args.password),
+        ])
+        service = self.client.get_service(project=self.get_project(), service=self.args.name)
+        with open(os.path.join(self.args.target_directory, "client.properties"), "w") as fp:
```
Comment (hnousiainen) [inline aiven/client/cli.py]: This is Kafka specific, should the command name then reflect that this is intended for Kafka?

Comment (ojarva) [reply aiven/client/cli.py]: Why is this Kafka specific? Doesn't the same apply for any other service with Java client and client certs, such as Cassandra beta?

Comment (hnousiainen) [reply aiven/client/cli.py]: Cassandra users password authentication, so at least the credentials are different. Truststore is probably generated for server side authentication.

client.properties is surely Kafka specific.

Comment (Ormod) [reply aiven/client/cli.py]: Updated name to be Kafka specific.

**Pull request merged**